jupyter-extension/src/plugins/mount/mountDrive.ts

```diff
@@ -123,6 +138,48 @@ export class MountDrive implements Contents.IDrive {
     };
   }
 
+  _fetchNextPage(
+    previousResponse: Contents.IModel,
+    key: string,
+    url: string,
+    getOptions: PartialJSONObject,
+  ): void {
+    // Stop fetching pages if we recieve a page less than expected file count of a page
+    if (previousResponse?.content?.length < PAGINATION_NUMBER) {
+      return;
+    }
+
+    // Queuing a promise without returning it to await lets us start the process of fetching the next pages in the background while
+    // continuing to render the the first page.
+    (async () => {
+      const nextResponse: Contents.IModel = await new Promise(
+        (resolve, reject) => {
+          return this._get(url, {
+            ...getOptions,
+            pagination_marker: previousResponse.content.slice(-1)[0].file_uri,
+          })
+            .then((nextResponse) => resolve(nextResponse))
+            .catch((e) => reject(e));
```

Comment:
I should show an error dialog for these errors like this PR does https://github.com/pachyderm/pachyderm/pull/9667/files#diff-945b8d32a095b3d419e4cd245d637ab5ca5282d16a2700a302d7493f75c8c091R115